packages/docs/PropTables/WorksheetPropTable.tsx

```diff
@@ -0,0 +1,289 @@
+import React from 'react';
+
+import { Code, NextLink, Prop, PropTable, PropTableWrapper } from '../components';
+
+const worksheetProps: Prop[] = [
+  {
+    name: 'columns',
+    types: ['TextColumn', 'NumberColumn', 'CheckboxColumn', 'SelectableColumn', 'ModalColumn'],
+    description: (
+      <>
+        See <NextLink href="#worksheet-text-column-prop-table">below</NextLink> for usage.
+      </>
+    ),
+    required: true,
+  },
+  {
+    name: 'items',
+    types: 'any[]',
+    description: 'The array of items to display.',
+    required: true,
+  },
+  {
+    name: 'onChange',
+    types: '(items: any[]) => void',
+    description: 'Returns the items that have been updated.',
+    required: true,
+  },
+  {
+    name: 'onErrors',
+    types: '(items: WorksheetError[]) => void',
+    description: (
+      <>
+        Returns an array of <NextLink href="#worksheet-error-prop-table">Error</NextLink> when an error is present.
+      </>
+    ),
+  },
+];
+
+const worksheetTextColumnProps: Prop[] = [
+  {
+    name: 'hash',
+    types: 'string',
+    required: true,
+    description: 'Unique identifier for column.',
+  },
+  {
+    name: 'header',
+    types: 'string',
+    required: true,
+    description: 'Header title.',
+  },
+  {
+    name: 'type',
+    types: ['text'],
```

Comment:
I would check some of these `types` key/values to see if you can just have them as a `string` instead of an `array`.
```suggestion
    types: 'text',
```